src/Equinor.ProCoSys.IPO.Infrastructure/Repositories/EventRepository.cs

```diff
@@ -0,0 +1,213 @@
+﻿using Equinor.ProCoSys.IPO.Command.Events;
+using Equinor.ProCoSys.IPO.MessageContracts;
+using System;
+using System.Linq;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Microsoft.EntityFrameworkCore;
+
+namespace Equinor.ProCoSys.IPO.Infrastructure.Repositories;
+
+public class EventRepository : RepositoryBase<Invitation>, IEventRepository
+{
+    public EventRepository(IPOContext context)
+        : base(context, context.Invitations,
+            context.Invitations
+                .Include(x => x.Participants)
+                .Include(x => x.McPkgs)
+                .Include(x => x.CommPkgs)
+                .Include(x => x.Comments))
+    {
+    }
+
+    public IInvitationEventV1 GetInvitationEvent(Guid invitationGuid)

```

Comment:
Name needs to reflect that we're only getting data from Local. 
This will only work if we already have Invitation loaded into memory.

I think I would reconsider the approach, we could just as attach all the data we need to the event when we create it?